extensions/elytron-security/deployment/src/test/java/io/quarkus/security/test/jdbc/SubjectExposingResource.java

```diff
@@ -0,0 +1,62 @@
+package io.quarkus.security.test.jdbc;
+
+import java.security.Principal;
+
+import javax.annotation.security.DenyAll;
+import javax.annotation.security.PermitAll;
+import javax.annotation.security.RolesAllowed;
+import javax.inject.Inject;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.core.Context;
+import javax.ws.rs.core.SecurityContext;
+
+/**
+ * @author Michal Szynkiewicz, michal.l.szynkiewicz@gmail.com
+ *         <br>
+ *         Date: 8/8/18
+ */
+@Path("subject")
+public class SubjectExposingResource {
```

Comment:
Why don't you reuse the existing `io.quarkus.security.test.SubjectExposingResource` ?